cmssw_known_errors.py

```diff
@@ -124,7 +124,7 @@
   "536.0": { "step": 1, "exitcode": 256,   "reason" : MSG_ASAN_INCOMPETIBILITY},
 }
 
-RelFilter="CMSSW_12_.+"
+RelFilter="CMSSW_(12|13)_.+"
```

Comment:
ah ok. I would suggest to change the regex so that it matches all releases 12 and above